packages/transformer/src/IModelTransformer.ts

```diff
@@ -369,6 +368,8 @@ type ChangeDataState =
  */
 export type InitFromExternalSourceAspectsArgs = InitOptions;
 
+type SyncType = "not-sync" | "forward" | "reverse";
```

Comment:
Do I have to export this type now that determineSyncType is a public static function which returns that type?

Reply:
hm nevermind, i made it so determineSyncType returns "forward" | "reverse", even though SyncType also has "not-sync". I feel like not-sync only applies to state determined within the transformer so I don't think it makes sense to return "not-sync" within the static function.